services/src/handlers/datasets.rs

```diff
@@ -135,60 +145,26 @@ async fn get_dataset_handler<C: Context>(
     Ok(web::Json(dataset))
 }
 
-/// Creates a new [Dataset](CreateDataset) using previously uploaded files.
+/// Creates a new dataset using previously uploaded files.
 /// Information about the file contents must be manually supplied.
-///
-/// # Example
-///
-/// ```text
-/// POST /dataset
-/// Authorization: Bearer fc9b5dc2-a1eb-400f-aeed-a7845d9935c9
-///
-/// {
-///   "upload": "420b06de-0a7e-45cb-9c1c-ea901b46ab69",
-///   "definition": {
-///     "properties": {
-///       "name": "Germany Border",
-///       "description": "The Outline of Germany",
-///       "sourceOperator": "OgrSource"
-///     },
-///     "metaData": {
-///       "OgrMetaData": {
-///         "loadingInfo": {
-///           "fileName": "germany_polygon.gpkg",
-///           "layerName": "test_germany",
-///           "dataType": "MultiPolygon",
-///           "time": "none",
-///           "columns": {
-///             "x": "",
-///             "y": null,
-///             "text": [],
-///             "float": [],
-///             "int": [],
-///             "bool": [],
-///             "datetime": [],
-///           },
-///           "forceOgrTimeFilter": false,
-///           "onError": "ignore"
-///         },
-///         "resultDescriptor": {
-///           "dataType": "MultiPolygon",
-///           "spatialReference": "EPSG:4326",
-///           "columns": {}
-///         }
-///       }
-///     }
-///   }
-/// }
-/// ```
-/// Response:
-/// ```text
-/// {
-///   "id": {
-///     "internal": "8d3471ab-fcf7-4c1b-bbc1-00477adf07c8"
-///   }
-/// }
-/// ```
+#[utoipa::path(
```

Comment:
If I try this method in swagger ui I get

>{
  "error": "BodyDeserializeError",
  "message": "missing field `type` at line 28 column 5"
}

Reply:
46ff795